docs/api.md

```diff
@@ -2830,6 +2913,56 @@ Identical to [the non-Draft version](/reference/submissions/attachments/clearing
 + Response 403 (application/json)
     + Attributes (Error 403)
 
+# Group Datasets
+
+_(introduced: version 2022.3)_
+
+A Dataset is a collection of Entities. A Dataset's schema is defined by the Forms that create Entities in that Dataset. See the [ODK XForms specification](https://getodk.github.io/xforms-spec) for defining Datasets in Form definitions.
```

Comment:
Feels like it needs to be fleshed out more and include some of the details that are mentioned in the create new form section about datasets. I'll work on this.